packages/yew/src/virtual_dom/vnode.rs

```diff
@@ -38,13 +44,49 @@ impl VNode {
             VNode::VText(_) => None,
             VNode::VPortal(vportal) => vportal.node.key(),
             VNode::VSuspense(vsuspense) => vsuspense.key.as_ref(),
+            VNode::VRaw(_) => None,
         }
     }
 
     /// Returns true if the [VNode] has a key.
     pub fn has_key(&self) -> bool {
         self.key().is_some()
     }
+
+    /// Create a [`VNode`] from a string of HTML
+    ///
+    /// # Behavior in browser
+    ///
+    /// In the browser, this function creates an element, sets the passed HTML to its `innerHTML`
+    /// and inserts the contents of it into the DOM.
+    ///
+    /// # Behavior on server
+    ///
+    /// When rendering on the server, the contents of HTML are directly injected into the HTML
+    /// stream.
+    ///
+    /// ## Warning
+    ///
+    /// The contents are **not** sanitized or validated. You, as the developer, are responsible to
+    /// ensure the HTML string passed to this method are _valid_ and _not malicious_
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # use yew::virtual_dom::VNode;
+    /// use yew::{html, AttrValue};
+    /// # fn _main() {
+    /// let parsed = VNode::from_raw_html(AttrValue::from("<div>content</div>"));
+    /// let _: VNode = html! {
+    ///     <div>
+    ///         {parsed}
+    ///     </div>
+    /// };
+    /// # }
```

Comment:
I think public documentation should use `Html` as an alias for VNode here.

Also, with the current behaviour, `<script>` tags are not executed correct? I think that's what setting `innerHtml` does, but I'm not sure. Since these tags would be executed when rendered with SSR, is that desired? Either way, I think mentioning that in the docs would be nice.

Reply:
It's mentioned:

> # Behavior in browser
> In the browser, this function creates an element, sets the passed HTML to its `innerHTML` and inserts the contents of it into the DOM.

Is that not enough?

Reply:
Yeah I guess for people who know what setting `innerHtml` implies it's enough. I was initially surprised that scripts were not executed in that case, so I thought repeating it here would be nice. But it's not a must, if you think it's well-known.